app/controllers/admin/staches_controller.rb

```diff
@@ -20,6 +20,12 @@ def index
       @staches = Stache.all.includes(:categories)
     end
 
+    def destroy
+      @stache = Stache.find(params[:id]).destroy
```

Comment:
I would probably put these on two separate lines for readability, but it's fine for now. Maybe when we do a little more refactoring, we can pull this `@stache = Stache.find(params[:id])` into a before_action.